{{cookiecutter.project_slug}}/Makefile

```diff
@@ -43,8 +43,8 @@ PRESET?=gpu-small
 HTTP_AUTH?=--http-auth
 
 # Command to run training inside the environment. Example:
-# TRAINING_COMMAND="bash -c 'cd $(PROJECT_PATH_ENV) && python -u $(CODE_DIR)/train.py --data $(DATA_DIR)'"
-TRAINING_COMMAND?='echo "Replace this placeholder with a training script execution"'
+TRAINING_COMMAND="bash -c 'cd $(PROJECT_PATH_ENV) && python -u $(CODE_DIR)/train.py --data $(DATA_DIR)'"
```

Comment:
how about we change to `TRAINING_COMMAND=python -u $(CODE_DIR)/train.py --data $(DATA_DIR)` and in the target, run:
```
train: _check_setup upload-code upload-config   ### Run a training job
	$(NEURO) run \
		--name $(TRAINING_JOB) \
		--preset $(PRESET) \
		--volume $(DATA_DIR_STORAGE):$(PROJECT_PATH_ENV)/$(DATA_DIR):ro \
		--volume $(PROJECT_PATH_STORAGE)/$(CODE_DIR):$(PROJECT_PATH_ENV)/$(CODE_DIR):ro \
		--volume $(PROJECT_PATH_STORAGE)/$(CONFIG_DIR):$(PROJECT_PATH_ENV)/$(CONFIG_DIR):ro \
		--volume $(PROJECT_PATH_STORAGE)/$(RESULTS_DIR):$(PROJECT_PATH_ENV)/$(RESULTS_DIR):rw \
		${OPTION_GCP_CREDENTIALS} \
		${OPTION_WANDB_CREDENTIALS} \
		--env PYTHONPATH=$(PROJECT_PATH_ENV) \
		--env EXPOSE_SSH=yes \
		--env JOB_TIMEOUT=0 \
		$(CUSTOM_ENV_NAME) \
		"bash -c 'cd $(PROJECT_PATH_ENV) && $(TRAINING_COMMAND)'"
```

Reply:
Yep, this makes sense to me.

Reply:
https://github.com/neuromation/cookiecutter-neuro-project/issues/282